@xen-orchestra/rest-api/src/open-api/oa-examples/user.oa-example.mts

```diff
@@ -0,0 +1,22 @@
+export const userIds = [
+  '/rest/v0/users/722d17b9-699b-49d2-8193-be1ac573d3de',
+  '/rest/v0/users/088124f3-41b6-4258-9653-6eedc7b46111',
+]
+
+export const partialUsers = [
+  {
+    permission: 'admin',
+    name: 'admin@admin.net',
+    id: '722d17b9-699b-49d2-8193-be1ac573d3de',
+    href: '/rest/v0/users/722d17b9-699b-49d2-8193-be1ac573d3de',
+  },
```

Comment:
Try to add another user in your example.
You should have the same number of result between `userIds` and `partialUsers`.